CMakeLists.txt

```diff
@@ -331,4 +380,12 @@ if (XEUS_R_BUILD_SHARED)
             DESTINATION ${XEUS_R_CMAKECONFIG_INSTALL_DIR})
 endif ()
 
+if(EMSCRIPTEN)
+    install(TARGETS xr
+            ARCHIVE DESTINATION ${XEUS_R_CMAKECONFIG_INSTALL_DIR})
```

Comment:
I don't think you need this if you install `xr.wasm` and `xr.js` below.

Reply:
(From in-person conversation with Johan): is there something else that this installs?

Reply:
Came from the cookie cutter I think 
https://github.com/jupyter-xeus/xeus-cookiecutter/blob/0cc37dcbd6c2ad2b250074b41a28cfeb716b4ac3/%7B%7Bcookiecutter.github_project_name%7D%7D/CMakeLists.txt#L365